README.md

```diff
@@ -16,3 +16,5 @@ Various methods to run the application locally are detailed below.
 Once up the application should be available on port 8101 - see the health page at http://localhost:8101/health.  
 
 Also try http://localhost:8101/swagger-ui.html to see the API specification.
+
+Set `spring.h2.console.enabled` to true in `application-dev-config.yml`. It is set to false in the repo because leaving it as true creates a security vulnerability in the cloud.
```

Comment:
there is no vulnerability - it is a false positive as the `dev` profile isn't used by any of our deployments.